plugin/template/README.md

```diff
@@ -24,9 +25,10 @@ template CLASS TYPE [ZONE...] {
 * **CLASS** the query class (usually IN or ANY).
 * **TYPE** the query type (A, PTR, ... can be ANY to match all types).
 * **ZONE** the zone scope(s) for this template. Defaults to the server zones.
-* **REGEX** [Go regexp](https://golang.org/pkg/regexp/) that are matched against the incoming question name. Specifying no regex matches everything (default: `.*`). First matching regex wins.
+* `match` **REGEX** [Go regexp](https://golang.org/pkg/regexp/) that are matched against the incoming question name. Omitting `match` will match everything (default: `.*`). First matching regex wins.
 * `answer|additional|authority` **RR** A [RFC 1035](https://tools.ietf.org/html/rfc1035#section-5) style resource record fragment
   built by a [Go template](https://golang.org/pkg/text/template/) that contains the reply.
+* `edns local` add an EDNS0 local OPT pseudo-RR to the response. Each **CODE**=**DATA** pair adds an EDNS0 local value.
```

Comment:
`local` is option, so **OPTION** or something, where `local` is the only one allowed?

Reply:
Yes, for now only local is allowed.  My intent was to make the syntax flexible enough to gracefully support adding other EDNS0 option types down the road.  Since the format after type would differ based on the type, IMO, even if we add more types, it would be easier to express the syntax for each supported type individually.

Reply:
ack, but where do I put those `CODE=DATA` values. I think this can be expanded upon  and made clearer on what keywords are and what the option are

Reply:
OK.  ... and an example would help ...

Reply:
After iterating on this a bit more, I think it would be clearer to make the syntax be  ... 

`edns-option CODE VALUE`

... multiple occurrences allowed, each occurrence adding an option code/value pair to the OPT record.

In the future, if there is a need for it, we could add an `edns-header` option that allows customization of other edns bits (i.e. extended rcode and other flags) ... e.g.  something like `edns-header X-RCODE DO` ... of which only one occurrence is allowed.

WDYT?